bindings/libdnf/rpm.i

```diff
@@ -20,6 +20,7 @@
     #include "libdnf/rpm/nevra.hpp"
     #include "libdnf/rpm/package.hpp"
     #include "libdnf/rpm/package_query.hpp"
+    #include "libdnf/rpm/package_sack.hpp"
```

Comment:
I see that the commit changes not only name of the class but also additional related parts. May I ask you for an improvement?